etl/steps/data/grapher/energy/2022-07-29/primary_energy_consumption.py

```diff
@@ -13,6 +13,9 @@ def run(dest_dir: str) -> None:
     garden_dataset = catalog.Dataset(DATASET_PATH)
     dataset = catalog.Dataset.create_empty(dest_dir, garden_dataset.metadata)
 
+    # Backward compatibility
+    dataset.metadata.title = "Primary energy consumption (BP & EIA, 2022a)"
```

Comment:
Maybe better call it "Primary energy consumption (BP & EIA, archive)", to avoid confusion (although this step should not be used to create a grapher dataset ever again).

Reply:
Good idea, thanks.